PythonTool/GraveIdentifyAndRemove.py

```diff
@@ -86,6 +86,10 @@ def remove_graveyard(input_file, output_file = None):
     ________
        output_file: str
            The name of the file written to the disk.
+
+    Raises:
+    _______
+       Exception: If no graveyard EntitySet is found.
```

Comment:
We'll probably want to include the type of exception here. (Lookup was a good choice btw)